client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/IsolationDatesFields.tsx

```diff
@@ -52,6 +52,7 @@ const IsolationDatesFields: React.FC<Props> = (props: Props): JSX.Element => {
                                     onBlur={props.onBlur}
                                     testId='quarantinedFromDate'
                                     labelText={errors[ClinicalDetailsFields.ISOLATION_START_DATE] ? errors[ClinicalDetailsFields.ISOLATION_START_DATE].message : '* מתאריך'}
+                                    helperText={null}
```

Comment:
why not just not pass helperText prop at all?

Reply:
because MUI forces to

Reply:
this is Meterial UI's datepicker not the component's :P see [this] (https://stackoverflow.com/questions/55734809/material-ui-pickers-disable-past-but-not-show-error) thread on stackoverflow